packages/mds-cache/redis/index.ts

```diff
@@ -15,11 +15,17 @@
  */
 
 import Redis, { KeyType, ValueType } from 'ioredis'
-import { Nullable, Timestamp } from '@mds-core/mds-types'
+import { Nullable, Timestamp, TimestampInSeconds } from '@mds-core/mds-types'
 import { isDefined, ClientDisconnectedError, ExceptionMessages } from '@mds-core/mds-utils'
 import { initClient } from './helpers/client'
 import { OrderedFields } from '../@types'
 
+export type ExpireAtOptions = {
+  key: KeyType
+  timeInSeconds?: TimestampInSeconds
+  timeInMs?: Timestamp
+}
+
```

Comment:
Only nit would be it may be worth adding a short comment to say what the precedence is here in the case that a user provides both, cause this isn't a union type of like `{ key: KeyType} & ({ timeInSeconds: TimestampInSeconds } | { timeInMs: Timestamp })` so the user _could_ provide options of `{ key: 'foo', timeInSeconds: 12345, timeInMs: 123456 }`

Or, you could add the comment on the `expireat` function if you think that we'll re-use this type elsewhere and have different precedence

Reply:
hard pass! looks like, but then _typeguards_ are required and sub-types, to make it usable. Alas, JS. 
Better comments tho.